docs/resource/README.md

```diff
@@ -77,7 +77,7 @@ as specified in the [Resource SDK specification](https://github.com/open-telemet
 
 **type:** `service`
 
-**Description:** A service instance.
+**Description:** A telemetry source. OpenTelemetry has adopted a broad interpretation such that every telemetry source is a service. Examples include, but are not limited to: web services, hosts, mobile applications, browser application, edge computing devices, functions as a service, databases, message brokers, etc. Specific types of telemetry sources may have additional conventions defining domain specific information, but the `service` conventions are applicable to all telemetry sources.
```

Comment:
What would qualify as telemetry source when scraping metrics from a Kubernetes cluster? Is it the cluster component responsible for creating and providing the metrics (the "source" in a technical sense)? Or is it a deployment, node, or container (the entity a piece of telemetry data primarily refers to, in a logical sense)?

Reply:
Its a good question and probably a key aspect of the design of the scraping component. The prometheus model (commonly used to scrape metrics from k8s clusters) every endpoint scraped is an "instance", and multiple replicated endpoints of an instance are a "job". These ideas map well to the idea of `service.name` (i.e. prometheus job) and `service.instance.id` (i.e. prometheus instance). The next question is can the scraper ascribe any other useful bits of information describing / identifying the telemetry source? I.e. does it know its scraping information about a node, deployment, or pod, or does everything role up into just a cluster? 

The idea of a common set of identifying resource attributes applicable across all telemetry sources works nicely with the #575 idea of adding a `service.type` attribute. With these in place, a OTLP receiver can reliably know the identity and type of telemetry producer. 

The next question to answer is how correlation works: If all telemetry sources have the same identifying attributes, how do we indicate that a service is running on a host in a cluster? We have attributes for a host and cluster, but when everything uses `service.*` for identity, we need conventions about how a host and a cluster populate the `service.*` attributes.

For example, consider a setup with a service running on a host. Both the service and host are telemetry producers:
```
// Host resource
{ service.name: "jberg", service.instance.id: "abc123", service.type: "host" }

// Service resource
{ service.name: "my-service", "service.instance.id: "def456", service.type: "web_service", host.name: "jberg", host.id: "abc123" }
```
If we have a convention that states that hosts set `service.name=${host.name}, service.instance.id=${host.id}, service.type=host`, then we can reliably known that "my-service" is running on the host because it has matching `host.name, host.id` resource attributes.

Reply:
I see, so `service.type` would identify the kind of entity that `service` attributes refer to.

However, it seems to me that with [its current proposed definition](https://github.com/open-telemetry/semantic-conventions/pull/575/files#diff-a8466589b56b8aefffe0f6d56a338bd7d9d71d3b36d43ca47e2d794adf5d0e34R110) it doesn't fully cover this use case:

> The `service.type` identifies the product that is deployed as the service.

A value like "host" doesn't match that description, the proposed definition of `service.type` would need to be generalized to account for that.

Reply:
Correct. This PR doesn't solve all the problems associated with identifying and correlating telemetry sources. It aims to codify just one concept (which I assert we already imply) - that all telemetry sources use the same attributes for identity - which we can build off of in future steps.